sdk/storage/storage-blob/ChangeLog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2019.11 12.0.0-preview.6
+
+- [Breaking] The default browser bundle has been removed from the npm package. Bundling your application with a bundler such as Webpack is the recommended approach to building a browser bundle.
```

Comment:
Also, point to the instructions in the readme ?

Reply:
This is a good idea, though I think we need sdk-wide instructions someplace that we can link to.